client/bootloader_flash.py

```diff
@@ -52,7 +52,7 @@ def flash_binary(fdesc, binary, base_address, device_class, destinations, page_s
     # First erase all pages
     for offset in range(0, len(binary), page_size):
         erase_command = commands.encode_erase_flash_page(base_address + offset, device_class)
-        utils.write_command(fdesc, erase_command, destinations)
+        utils.write_command_retry(fdesc, erase_command, destinations)
```

Comment:
> Something like abort ?

Yes, a failed command (wrong device class, invalid address) should result in an abort.